plugins/point.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: point
+spec:
+  version: "v0.1.0"
+  homepage: https://github.com/davidchua/kubectl-point
+  shortDescription: "Creates a headless service to an external source with optional ingress"
```

Comment:
Just fyi this probably won't fit where it's shown and will be truncated. Limit to 50 chars if you can.